life_expectancy/cleaning.py

```diff
@@ -1,38 +1,33 @@
+""" Function for clean data """
 import argparse
 import pandas as pd
 
-def load_data(path):
-    """Load data from a TSV file."""
-    df = pd.read_csv(path, sep="\t")
-    df.columns = [col.replace("\\", "") for col in df.columns]
-    return df
+def clean_data(country="PT"):
+    """Function to load and clean the tsv file for a specific country.
+
+    Returns dataframe cleaned and saved as csv
+    """
+    df = pd.read_csv("life_expectancy/data/eu_life_expectancy_raw.tsv", sep="\t")
```

Comment:
Question: I though we were supposed to create a function to load the data and separated from the main function?